docs/running_Occlum_with_Docker_and_OCI_Runtime_rune.md

```diff
@@ -0,0 +1,140 @@
+# Running Occlum with Docker and OCI Runtime rune
+
+This user guide provides the steps to run Occlum with Docker and OCI Runtime `rune`.
+
+[rune](https://github.com/alibaba/inclavare-containers/tree/master/rune) is a novel OCI Runtime used to run trusted applications in containers with the hardware-assisted enclave technology.
+
+[Occlum](https://github.com/occlum/occlum) is a memory-safe, multi-process library OS for Intel SGX.
+
+# Requirements
+
+- Ensure that you have one of the following required operating systems to build a Occlum container image:
+  - CentOS 8.1
+  - Ubuntu 18.04-server
+
+- Please follow [Intel SGX Installation Guide](https://download.01.org/intel-sgx/sgx-linux/2.11/docs/Intel_SGX_Installation_Guide_Linux_2.11_Open_Source.pdf) to install Intel SGX driver, Intel SGX SDK & PSW for Linux.
+  - For CentOS 8.1, UAE service libraries are needed but may not installed if SGX PSW installer is used. Please manually install it:
+    ```shell
+    rpm -i libsgx-uae-service-2.11.100.2-1.el8.x86_64.rpm
+    ```
+
+- Install [enable_rdfsbase kernel module](https://github.com/occlum/enable_rdfsbase#how-to-build), allowing to use FSGSBASE instructions in Occlum.
+
+- Install rune and occlum.
+  - For CentOS 8.1:
+    1. Add the repository to your sources.
+    ```shell
+    cat >/etc/yum.repos.d/inclavare-containers.repo <<EOF
+    [inclavare-containers]
+    name=inclavare-containers
+    enabled=1
+    baseurl=https://mirrors.openanolis.org/inclavare-containers/rpm-repo/
+    gpgcheck=1
+    repo_gpgcheck=1
+    gpgkey=https://mirrors.openanolis.org/inclavare-containers/rpm-repo/RPM-GPG-KEY-rpm-sign
+    gpgcakey=https://mirrors.openanolis.org/inclavare-containers/rpm-repo/RPM-GPG-KEY-rpm-sign-ca
+    EOF
+    ```
+
+    2. Install the RPM packages.
+    ```shell
+    sudo yum install -y rune occlum
+    source /etc/profile
+    ```    
+
+  - For Ubuntu 18.04-server:
+    1. Add the repository to your sources.
+    ```shell
+    echo 'deb [arch=amd64] https://mirrors.openanolis.org/inclavare-containers/deb-repo bionic main' | tee /etc/apt/sources.list.d/inclavare-containers.list
+    ```
+
+    2. Add the key to the list of trusted keys used by the apt to authenticate packages.
+    ```shell
+    wget -qO - https://mirrors.openanolis.org/inclavare-containers/deb-repo/DEB-GPG-KEY.key | sudo apt-key add -
+    ```
+
+    3. Update the apt and install the packages.
+    ```shell
+    sudo apt-get update
+    sudo apt-get install -y rune occlum
+    source /etc/profile
+    ```
+
+# Building Occlum container image
+
+## Prepare "hello world" demo program
+
+[This tutorial](https://github.com/occlum/occlum#hello-occlum) can help you to create your first occlum build.
+
+Assuming "hello world" demo program in `occlum_instance` directory is built.
```

Comment:
Occlum has a new command "occlum package". The "package" command helps user create a occlum instance package. It is better than copy the entire folder.